.github/workflows/workflow.claude.yml

```
@@ -413,3 +413,112 @@ jobs:
+
+      - name: Post analysis to issue
+        env:
+          GH_TOKEN: ${{ steps.app-token.outputs.token }}
+        run: |
+          if [ -f /tmp/analysis-output.md ] && [ -s /tmp/analysis-output.md ]; then
+            gh issue comment ${{ github.event.issue.number }} --body-file /tmp/analysis-output.md
+          else
+            gh issue comment ${{ github.event.issue.number }} --body-file - <<EOF
+## Requirements Analysis: Issue #${{ github.event.issue.number }}
+
+No conflicts or significant impact detected.
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)